content/en/docs/instrumentation/erlang/getting-started.md

```diff
@@ -287,7 +287,7 @@ when rendered in your tracing tool of choice.
 ### Next Steps
 
 Enrich your automatically generated instrumentation with
-[manual instrumentation](/docs/instrumentation/erlang/instrumentation) of your
+[manual instrumentation](/docs/instrumentation/erlang/manual) of your
 own codebase. This allows you to customize the observability data your
 application emits.
```

Comment:
due to indentation text width, this now messes with the prettifier.

```suggestion
290,292c290,292
[manual instrumentation](/docs/instrumentation/erlang/manual) of your own
codebase. This allows you to customize the observability data your application
emits.
```

based on the prettifier diff being:

```diff
290,292c290,292
< [manual instrumentation](/docs/instrumentation/erlang/manual) of your
< own codebase. This allows you to customize the observability data your
< application emits.
---
> [manual instrumentation](/docs/instrumentation/erlang/manual) of your own
> codebase. This allows you to customize the observability data your application
> emits.
```

Reply:
Thank you @ferd !